modules/t/xref_parser.t

```diff
@@ -364,8 +364,14 @@ sub test_refseq {
     dependent_xref => 1,
   }, "$type RefSeq entries hang off INSDC entries", tmp_file_name => $file);
 }
-test_refseq($refseq_protein_elegans_record, "protein");
-test_refseq($refseq_mrna_elegans_record, "mrna");
+
+# FIXME
+# silenced due to refactoring of XrefParser::RefSeqCoordinateParser
+# pending updates to XrefParser::WormbaseCElegansRefSeqGPFFParser
+
+# test_refseq($refseq_protein_elegans_record, "protein");
```

Comment:
The tests pass locally if I merge https://github.com/Ensembl/ensembl/pull/337/commits/7c80847980e51243cf0ed409c404193ed0410be0 and do this:
```
sub test_refseq {
   my ($record, $type) = @_;
-  my $file = "something_that_says_$type";
+  my $file = "RefSeq_$type";
   test_parser("XrefParser::RefSeqGPFFParser",$record, {
```
Thanks to you keeping the interfaces I think it should all still work!

Reply:
reopened some of the tests that now pass, some continue off
take a look at them to see what can be done